.github/workflows/add-lockdown-label.yml

```diff
@@ -0,0 +1,78 @@
+name: Add Branch Lockdown Label to PRs
+
+on:
+  schedule:
+    # Run daily at 00:00 UTC
+    - cron: '0 23 * * *'
```

Comment:
I'm not convinced this is the right approach overall. The reason is, this runs once a day. Standard label bots run as **PRs are created**. That's what triggers them to run. Sure, this process will work on pre-existing PRs just fine. But as things attempt to "flow" from other repos during the day, domestic cat might accidentally merge things they shouldn't.

Also, triggering this at the "start of the day" like this, will there be any issues with the date calculation below? Most processes like this avoid the exact start of a day because stuff like daylight savings time exists. Plus, this runs right in the middle of Jason's shift, I believe.

Reply:
So you think it should trigger on every PR do these branches then? That means it runs a lot of times but maybe that's ok. I wonder then if it runs on PRs, if we could just run it against the PR that's being created rather than querying all the PRs.

Reply:
I'm fairly certain you can trigger on PR creation. But I don't know GitHub Actions.

Reply:
Yes, you can. I believe you can also trigger it on other PR events (opened, closed, assigned, etc...)
https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request